packages/design-system-react/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../../tsconfig.packages.json",
   "compilerOptions": {
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "jsx": "react"
```

Comment:
Cannot use JSX without this setting

<img width="635" alt="Screenshot 2024-10-30 at 2 38 01 PM" src="https://github.com/user-attachments/assets/92a4d8da-66d8-40a5-bfdb-0ee3fc6da916">